src/components/RequestInviteForm.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getAuthMemberData } from '../connectors';
+import { getRequestInviteOperation } from '../operations';
+import { postOperation } from '../actions';
+import YoutubeVideo from './YoutubeVideo';
+
+interface Props {
+  toUid: string;
+  toMid: string;
+  fullName: string;
+  videoUrl: string;
+}
+
+class RequestInviteForm extends Component<Props> {
+    constructor(props: InviteState) {
+      super(props);
+      this.state = {fullName: "", videoUrl: "", errorMessage: ""};
+    }
+
+    setFullName(event: any) : void {
+      this.setState({ fullName: event.target.value })
+    }
+
+    setVideoURL(event: any) : void {
+      this.setState({ videoUrl: event.target.value })
+    }
+
+    handleOnSubmit(event: any) : void {
+      event.stopPropagation();
+
+      if (this.state.fullName.length <= 0) {
+        this.setState({ errorMessage: "Please enter you name" });
+      }
+
+      else if (this.state.videoUrl.length <= 0) {
+        this.setState({ errorMessage: "Please enter a valid url" });
+      }
+
+      else {
+        let requestOp = getRequestInviteOperation(
```

Comment:
@markulrich wanted to get some feedback - is this the correct way to go about making a requestInviteOp?

Reply:
Yes that's great!